File: 0101-0200/0168.Excel_Sheet_Column_Title.java
//0ms





class Solution {
    public String convertToTitle(int columnNumber) {
        StringBuilder title=new StringBuilder();
        while(columnNumber>0){
            columnNumber--;
            title.insert(0,(char)(columnNumber%26+'A'));
            columnNumber=columnNumber/26;
        }
        return title.toString();
    }
}




//0ms





class Solution {
    public String convertToTitle(int col) {

        StringBuilder sb = new StringBuilder();


        while(col>0){
            col--;
            char c = (char) ('A' + col%26);
        sb.append(c);
            col = col / 26;
        }
        
        return sb.reverse().toString();


        
    }
}

File: 0101-0200/0168.Excel_Sheet_Column_Title.cpp
//0ms






class Solution {
public:
    string convertToTitle(int columnNumber) {
        string res;
       while(columnNumber>0){
        columnNumber--;
             int k=columnNumber%26;
             columnNumber/=26;
             res+=char('A'+k);
       }
       reverse(res.begin(),res.end());
       return res;
    }
};






//0ms






class Solution {
public:
    string convertToTitle(int columnNumber) {
        string res="";
        while(columnNumber>0)
        {
            columnNumber--;
            res=char(columnNumber%26+'A')+res;
            columnNumber/=26;
        }
        return res;
    }
};
